frontend/src/app/general/header/header.component.html

```diff
@@ -3,83 +3,91 @@
-          target="_blank"
-          >Open Issue on Github <mat-icon>open_in_new</mat-icon>
-        </a>
-      </div>
-      <mat-menu #profileMenu="matMenu">
-        <a
-          *ngIf="userService.user?.role === 'administrator'"
-          class="profileMenuButton"
-          mat-menu-item
-          routerLink="settings"
-        >
-          Settings <mat-icon>settings</mat-icon>
-        </a>
-        <a
-          *ngIf="userService.user?.role === 'administrator'"
-          class="profileMenuButton"
-          mat-menu-item
-          routerLink="events"
+          *ngIf="userService.validateUserRole(item.requiredRole)"
+          [attr.href]="item.href"
+          [attr.target]="item.target"
         >
-          Events <mat-icon>event_note</mat-icon>
+          {{ item.name }} <mat-icon *ngIf="item.icon">{{ item.icon }}</mat-icon>
         </a>
-        <a
-          class="profileMenuButton"
-          mat-menu-item
-          routerLink="logout/redirect"
-          [queryParams]="{ reason: 'logout' }"
-          *ngIf="authService.isLoggedIn()"
+      </div>
+    </div>
+    <div>
+      <div class="hidden xl:flex xl:right-aligned gap-2">
+        <div>
+          <a
+            mat-raised-button
+            color="primary"
+            href="https://github.com/DSD-DBS/capella-collab-manager/issues"
+            target="_blank"
+            >Open Issue on Github <mat-icon>open_in_new</mat-icon>
+          </a>
+        </div>
+        <mat-menu #profileMenu="matMenu">
+          <a
+            *ngIf="userService.user?.role === 'administrator'"
+            class="profileMenuButton"
+            mat-menu-item
+            routerLink="settings"
+          >
+            Settings <mat-icon>settings</mat-icon>
+          </a>
+          <a
+            *ngIf="userService.user?.role === 'administrator'"
+            class="profileMenuButton"
+            mat-menu-item
+            routerLink="events"
+          >
+            Events <mat-icon>event_note</mat-icon>
+          </a>
+          <a
+            class="profileMenuButton"
+            mat-menu-item
+            routerLink="logout/redirect"
+            [queryParams]="{ reason: 'logout' }"
+            *ngIf="authService.isLoggedIn()"
+          >
+            Log out <mat-icon>logout</mat-icon>
+          </a>
+        </mat-menu>
+
+        <button
+          class="!ml-5 !mr-5"
+          color="primary"
+          mat-raised-button
+          [matMenuTriggerFor]="profileMenu"
         >
-          Log out <mat-icon>logout</mat-icon>
-        </a>
-      </mat-menu>
-      <button
-        color="primary"
-        id="profile"
-        mat-raised-button
-        [matMenuTriggerFor]="profileMenu"
-      >
-        Profile <mat-icon>account_circle</mat-icon>
-      </button>
+          Profile <mat-icon>account_circle</mat-icon>
+        </button>
+      </div>
     </div>
   </div>
+
+  <div class="primaryText ml-5 select-none text-base">
```

Comment:
```suggestion
  <div class="primaryText ml-5 pb-3 select-none text-base">
```

Add a padding to the bottom, otherwise the header ends directly with the breadcrumb, which looks unevenly spaced.